examples/DotNetCore/SimpleConsoleApp/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.SimpleConsoleApp
+{
+    using Microsoft.Extensions.Configuration;
+
+    class Program
+    {
+        static void Main(string[] args) {
+
+            var builder = new ConfigurationBuilder();
+            builder.AddAzureAppConfiguration(Environment.GetEnvironmentVariable("ConnectionString"));
+
+            var config = builder.Build();
+            Console.WriteLine(config["TestApp:Settings:Message"] ?? "Hello world!");
```

Comment:
Instead of "Hello world!", I would put something like below if the key is not found.

$"Please create a key-value with the key '{key}' in Azure App Configuration."

Reply:
Sonds great! Updated.